web/src/beta/features/Editor/tabs/map/LeftPanel/Layers/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { useReactiveVar } from "@apollo/client";
+import React from "react";
+
+import Icon, { Icons } from "@reearth/beta/components/Icon";
+import * as Popover from "@reearth/beta/components/Popover";
+import PopoverMenuContent from "@reearth/beta/components/PopoverMenuContent";
+import Text from "@reearth/beta/components/Text";
+import type { NLSLayer } from "@reearth/services/api/layersApi/utils";
+import { showDataSourceManagerVar, showPopOverLayerButtonVar } from "@reearth/services/state";
+import { styled } from "@reearth/services/theme";
+
+type LayersProps = {
+  layers: NLSLayer[];
+  onLayerDelete: (id: string) => void;
+  onLayerSelect: (id: string) => void; // Todo
+};
+
+const Layers: React.FC<LayersProps> = ({ layers, onLayerDelete }) => {
+  const [addMenuOpen, setAddMenuOpen] = React.useState(false);
+
+  const showPopOverLayerButton = useReactiveVar(showPopOverLayerButtonVar);
+
+  return (
+    <StyledLayerContainer>
+      <Popover.Provider
+        open={addMenuOpen}
+        onOpenChange={() => setAddMenuOpen(s => !s)}
+        placement="bottom-end">
+        <Popover.Trigger asChild>
+          <StyledAddLayerIcon onClick={() => setAddMenuOpen(true)}>
+            <Icon icon="addLayer" />
+          </StyledAddLayerIcon>
+        </Popover.Trigger>
+        {showPopOverLayerButton && (
+          <Popover.Content>
+            <PopoverMenuContent
+              size="md"
+              items={[
+                {
+                  name: "Add Layer from Resource",
+                  icon: "file",
+                  onClick: () => {
+                    showDataSourceManagerVar(true);
+                    showPopOverLayerButtonVar(false);
+                  },
+                },
+                {
+                  name: "Add Sketch Layer",
+                  icon: "pencilSimple",
+                  onClick: () => {},
+                },
+              ]}
+            />
+          </Popover.Content>
+        )}
+      </Popover.Provider>
+      {layers.map(l => (
+        <ListItem
+          key={l.id}
+          item={l.title}
+          menu={[
+            {
+              name: "Delete",
+              icon: "bin",
+              onClick: () => onLayerDelete(l.id),
+            },
+          ]}
+        />
+      ))}
+    </StyledLayerContainer>
+  );
+};
+
+// utils
```

Comment:
Not sure what this means, but either way remove it. The name of the component should be enough without a comment

Reply:
Done.